src/Wallabag/ApiBundle/Controller/WallabagRestController.php

```diff
@@ -138,6 +139,30 @@ public function getEntryAction(Entry $entry)
     }
 
     /**
+     * Retrieve a single entry as a predefined format.
+     *
+     * @ApiDoc(
+     *      requirements={
+     *          {"name"="entry", "dataType"="integer", "requirement"="\w+", "description"="The entry ID"}
+     *      }
+     * )
+     *
+     * @Route(requirements={"_format"="epub|mobi|pdf|txt|csv"})
```

Comment:
It lacks xml export, but this should be accessible directly through the regular api calls ?